contracts/src/bridge/SequencerInbox.sol

```diff
@@ -124,22 +200,13 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
             _totalDelayedMessagesRead
         );
         uint256 __totalDelayedMessagesRead = _totalDelayedMessagesRead;
-        uint256 prevSeqMsgCount = bridge.sequencerReportedSubMessageCount();
-        uint256 newSeqMsgCount = prevSeqMsgCount +
-            _totalDelayedMessagesRead -
-            totalDelayedMessagesRead;
+        // this skips `validateBatchData` since no sequencer batch is actually added here
         (
             uint256 seqMessageIndex,
             bytes32 beforeAcc,
             bytes32 delayedAcc,
             bytes32 afterAcc
-        ) = addSequencerL2BatchImpl(
-                dataHash,
-                __totalDelayedMessagesRead,
-                0,
-                prevSeqMsgCount,
-                newSeqMsgCount
-            );
+        ) = addSequencerL2BatchImpl(dataHash, __totalDelayedMessagesRead, 0);
```

Comment:
We'll want to update the sequencer message count as we did before, because it means that the batch poster can still function after a force include.